src/main/java/org/kainos/ea/db/DbConnector.java

```diff
@@ -0,0 +1,43 @@
+package org.kainos.ea.db;
+import java.io.FileInputStream;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class DbConnector {
+    private static Connection conn;
+
+    public static Connection getConnection() throws SQLException{
+        String user, password, host, name;
+
+        if (conn != null && !conn.isClosed())
+        {
+            return conn;
+        }
+        try {
+            FileInputStream propsStream = new FileInputStream("~/db.properties");
```

Comment:
This is looking for a db.properties file in your Home directory, instead of a relative path within the project. Is this what you were after

Reply:
changed on new commit